rosetta-source/src/main/rosetta/product-qualification-func.rosetta

```diff
@@ -274,16 +276,17 @@ func Qualify_UnderlierProduct_Equity: <"Qualifies a product as having the Asset
             or (underlier -> security -> securityType = SecurityTypeEnum -> Fund
                 and underlier -> security -> fundType = FundProductTypeEnum -> MutualFund)
             or underlier -> security -> securityType = SecurityTypeEnum -> Warrant
-            or (underlier -> index exists /* and underlier -> index -> indexType = IndexTypeEnum -> Equity */ ) // No FpML mappings exist for IndexTypeEnum
-                // Qualifies that the underlier is a basket composed of equity products only
+            or underlier -> index ->> assetClass = AssetClassEnum -> Equity
+            // Qualifies that the underlier is a basket composed of equity products only
```

Comment:
All the assertions below for equity-only basket are duplicates of the single-name assertions. Consider factoring into a function (that takes `Security` as input and returns a boolean) and invoke in the single-name and basket tests.

Reply:
P3